test/module/irohad/validation/query_execution.cpp

```diff
@@ -453,19 +460,22 @@ TEST_F(GetAccountAssetsTest, DomainAccountValidCase) {
       .WillOnce(Return(admin_roles));
   EXPECT_CALL(*wsv_query, getRolePermissions(admin_role))
       .WillOnce(Return(role_permissions));
-  EXPECT_CALL(*wsv_query, getAccountAsset(account_id, asset_id))
-      .WillOnce(Return(accountAsset));
+  EXPECT_CALL(*wsv_query, getAccountAssets(account_id))
+      .WillOnce(Return(std::vector<std::shared_ptr<shared_model::interface::AccountAsset>>(
+          {accountAsset})));
 
   auto response = validateAndExecute(query);
   ASSERT_NO_THROW({
-    auto &cast_resp = boost::apply_visitor(
-        shared_model::interface::SpecifiedVisitor<
-            shared_model::interface::AccountAssetResponse>(),
-        response->get());
-
-    ASSERT_EQ(cast_resp.accountAsset().accountId(), account_id);
-    ASSERT_EQ(cast_resp.accountAsset().assetId(), asset_id);
-  });
+                    auto &cast_resp = boost::apply_visitor(
```

Comment:
IMO it does not make any sense to wrap it with ASSERT_NO_THROW. Instead cast_rep should be checked `ASSERT_TRUE(cast_rep)`.